### PR TITLE
Refine some text regarding the use of encryption

### PR DESCRIPTION
**Pull request by robUx4**
All text suggested by Roman Danyliw in https://mailarchive.ietf.org/arch/msg/cellar/E6iB8RosCz5qjvx8IJkcCdtMfGw/

### CODE REVIEW

File: notes.md [hunk 1]
```diff
@@ -675,13 +675,17 @@ Country codes are the [@!BCP47] two-letter region subtag, without the UK excepti
...
-encryption framework in Matroska as a type of DRM (Digital Rights Management).
-
-This document does not specify any kind of standard for encrypting elements.
-The issue of key scheduling, authorisation, and authentication are out of scope.
-External entities have used these elements in proprietary ways.
+This Matroska specification provides no interoperable solution for securing the
+data container with any assurances of confidentiality, integrity, authenticity,
+or to provide authorization. The `ContentEncryption Element` ((#contentencryption-element))
+and associated sub-fields ((#contentencalgo-element) to (#aessettingsciphermode-element)) are defined
+only for the benefit of implementers to construct their own propriety solution
```
Comment (retokromer) [inline notes.md:682]: proprietary?

Comment (robUx4) [reply notes.md]: Good catch!

**Review by retokromer — APPROVED** [review]

File: notes.md [hunk 2]
```diff
@@ -675,13 +675,17 @@ Country codes are the [@!BCP47] two-letter region subtag, without the UK excepti
...
+This Matroska specification provides no interoperable solution for securing the
+data container with any assurances of confidentiality, integrity, authenticity,
+or to provide authorization. The `ContentEncryption Element` ((#contentencryption-element))
+and associated sub-fields ((#contentencalgo-element) to (#aessettingsciphermode-element)) are defined
+only for the benefit of implementers to construct their own proprietary solution
+or as the basis for further standardization activities.  How to use these
+fields to secure a Matroska data container is out of scope.  As are any related
+issues such as key management and distribution.
+
+A `Matroska Reader` who encounter containers that use the fields defined in this
```
Comment (mbunkus) [inline notes.md:687]: "A reader" is singular, isn't it? Should be "encounters", then.

File: notes.md [hunk 3]
```diff
@@ -696,12 +700,22 @@ Encryption information is stored in the `ContentEncodings Element` under the `Co
...
 The `ContentEncodingScope Element` gives an idea of which part of the track are encrypted.
 But each `ContentEncAlgo Element` and its sub elements like `AESSettingsCipherMode` really
 define how the encrypted should be exactly interpreted.
 
-The AES-CTR system, which corresponds to `ContentEncAlgo` = 5 ((#contentencalgo-element)) and `AESSettingsCipherMode` = 1 ((#aessettingsciphermode-element)),
-is defined in the [@?WebM-Enc] document.
+An example of an extension that builds upon these security-related fields in this specification is [@?WebM-Enc].
+It uses AES-CTR, `ContentEncAlgo` = 5 ((#contentencalgo-element)) and `AESSettingsCipherMode` = 1 ((#aessettingsciphermode-element)).
+
+Matroska Writers **MUST NOT** use insecure cryptographic algorithms to create new
```
Comment (mbunkus) [inline notes.md:716]: Isn't "Matroska Writer" the same kind of term as "`Matroska Reader`"? Shouldn't it be `Matroska Writer` (backticked)?

**Review by mbunkus — CHANGES_REQUESTED** [review]

File: notes.md (hunk 3 above)
Comment (robUx4) [reply notes.md:716]: It's consistent throughout the whole document, but yes it's one of the terms that should be backticked.

File: notes.md [hunk 4]
```diff
@@ -675,13 +675,17 @@ Country codes are the [@!BCP47] two-letter region subtag, without the UK excepti
...
-This document does not specify any kind of standard for encrypting elements.
-The issue of key scheduling, authorisation, and authentication are out of scope.
-External entities have used these elements in proprietary ways.
+This Matroska specification provides no interoperable solution for securing the
+data container with any assurances of confidentiality, integrity, authenticity,
+or to provide authorization. The `ContentEncryption Element` ((#contentencryption-element))
+and associated sub-fields ((#contentencalgo-element) to (#aessettingsciphermode-element)) are defined
+only for the benefit of implementers to construct their own proprietary solution
+or as the basis for further standardization activities.  How to use these
+fields to secure a Matroska data container is out of scope.  As are any related
```
Comment (retokromer) [inline notes.md:684]: I would use a comma and only one sentence: `is out of scope, as are any`. But that’s more a matter of personal taste …

Comment (robUx4) [reply notes.md]: You're right.

**Review by mbunkus — APPROVED** [review]
Thanks!

**Review by retokromer — APPROVED** [review]

**Pull request merged**